src/dos/dos_misc.cpp

```diff
@@ -63,6 +63,10 @@ static Bitu INT2A_Handler(void) {
 
 static bool DOS_MultiplexFunctions(void) {
 	switch (reg_ax) {
+	case 0x1000:
```

Comment:
Maybe document what `0x1000` stands for?

Reply:
I put a comment directly under that line.  This is an interrupt function and all it does is basically return "true" if `SHARE.EXE` is installed.  I don't know that it even has a documented name.  This is the way all interrupt functions look in our codebase.